From 87b2a3e49be92365b4084d5aec064ba72cb667e4 Mon Sep 17 00:00:00 2001
From: Sunil Goutham <sgoutham@marvell.com>
Date: Wed, 21 Aug 2019 22:06:18 +0530
Subject: [PATCH 0272/1921] octeontx2-af: Fix compilation issue

Fix compilation issue introduced by patch
'commit e833c99ef4dd ("octeontx2-af: Transmit packets during SMQ flush")'

Change-Id: Ice1473e779d04263577e73d5e83e2d925642cdc9
Signed-off-by: Sunil Goutham <sgoutham@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/14606
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Reviewed-on: https://sj1git1.cavium.com/14930
Tested-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/af/rvu_fixes.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/drivers/net/ethernet/marvell/octeontx2/af/rvu_fixes.c b/drivers/net/ethernet/marvell/octeontx2/af/rvu_fixes.c
index e9e9c07335c2..5dac149dd1e0 100644
--- a/drivers/net/ethernet/marvell/octeontx2/af/rvu_fixes.c
+++ b/drivers/net/ethernet/marvell/octeontx2/af/rvu_fixes.c
@@ -999,6 +999,11 @@ bool is_parse_nibble_config_valid(struct rvu *rvu,
 	return true;
 }
 
+void __weak otx2smqvf_xmit(void)
+{
+	/* Nothing to do */
+}
+
 void rvu_smqvf_xmit(struct rvu *rvu)
 {
 	if (is_rvu_95xx_A0(rvu) || is_rvu_96xx_A0(rvu)) {
-- 
2.31.1

